src/grpc/grpc_server.h

```diff
@@ -100,6 +113,12 @@ class Server {
       const std::shared_ptr<SharedMemoryManager>& shm_manager,
       const Options& server_options, std::unique_ptr<Server>* server);
 
+  static bool CreateWrapper(
```

Comment:
Function was called `CreateWrapper()` to avoid confusion with the original and primary `Create()` function used by `main.cc`. But, I just changed the function name to `Create()` through overloading.

boolean was returned from `tritonfrontend.h` for simplicity of getting something working. 
Latest commit should reflect the expected flow for error handling in bindings consistent with the core repo.